superset-frontend/src/explore/main.less

```diff
@@ -106,14 +98,16 @@
   float: left;
   padding-right: 3px;
 
-  .Select-control {
+  .dropdown {
     display: flex;
-
-    .Select-arrow-zone {
-      width: auto;
-      position: absolute;
-      right: 0;
-      top: 12px;
+    button {
+      padding-right: 20px;
+      .caret {
+        width: auto;
+        position: absolute;
+        right: 5px;
+        top: 6px;
+      }
```

Comment:
This is for fixing some styles in date selector:

### Before

![image](https://user-images.githubusercontent.com/335541/81363494-5133cf00-9098-11ea-8d98-1f711b1aef55.png)


### After

![image](https://user-images.githubusercontent.com/335541/81363549-745e7e80-9098-11ea-8ca4-9eb5393b0532.png)